http_request_randomizer/requests/parsers/SamairProxyParser.py

```diff
@@ -46,6 +47,15 @@ def parse_proxyList(self):
             # Make sure it is a Valid Proxy Address
             if UrlParser.valid_ip_port(td_row.text):
                 curr_proxy_list.append('http://' +td_row.text)
+                proxy_object = ProxyObject()
```

Comment:
Looks good, we probably want to make the code more clear and move this chunk of code in a separate method which returns a ProxyObject